### PR TITLE
fix: replace touchactions with w3c action chains

### LINKED ISSUE #40: Update swipe coordinates methods
right now the methods for swiping are getting deprecated because it is using Appium TouchAction:

```
 Appium::Core::TouchAction is deprecated in W3C spec. Use W3C actions instead
```

https://www.rubydoc.info/github/appium/ruby_lib_core/Appium/Core/TouchAction

### PR DESCRIPTION
**Pull request by TDL-EdgarsEglitis**
Fixes #40

### CODE REVIEW

File: lib/core/device.rb [hunk 1]
```diff
@@ -701,36 +689,24 @@ def driver_method(action)
...
+    start_x = action["StartX"]
+    start_y = action["StartY"]
+    end_x = action["EndX"] ? action["EndX"] : 0
+    end_y = action["EndY"] ? action["EndY"] : 0
 
-    action = Appium::TouchAction.new(@driver).swipe(opts).perform
+    @driver.action
+      .move_to_location(start_x, start_y)
+      .pointer_down(:left)
+      .move_to_location(end_x, end_y, duration: 200)
```
Comment (alvarolaserna) [inline lib/core/device.rb:707]: same

**Review by alvarolaserna — CHANGES_REQUESTED** [review]

File: lib/core/device.rb [hunk 2]
```diff
@@ -649,13 +641,13 @@ def clear_field(action)
...
-      start_y: el_location.y,
-      end_x: el_location.x,
-      end_y: 0,
-    }
-    action = Appium::TouchAction.new(@driver).swipe(opts).perform
+
+    @driver.action
+      .move_to_location(el_location.x, el_location.y)
+      .pointer_down(:left)
+      .move_to_location(el_location.x, 0, duration: 200)
```
[the remark this answers is not in the document]
Comment (TDL-EdgarsEglitis) [reply lib/core/device.rb]: Good catch, I checked Selenium's source code and it is indeed treated as a value in seconds. Fixed all 4 instances just now.

File: lib/core/device.rb [hunk 3]
```diff
@@ -683,15 +673,13 @@ def swipe_elements(action)
...
-      end_x: el2_x,
-      end_y: el2_y,
-      duration: 2000,
-    }
 
-    action = Appium::TouchAction.new(@driver).swipe(opts).perform
+    @driver.action
+      .move_to_location(el1_x, el1_y)
+      .pointer_down(:left)
+      .move_to_location(el2_x, el2_y, duration: 2)
```
Comment (alvarolaserna) [inline lib/core/device.rb:680]: why have you put 2 for this and 0.2 for the rest? 2 seconds sounds like a long swipe imo

Comment (TDL-EdgarsEglitis) [reply lib/core/device.rb:680]: This matches the previously configured value of 2000. All other places were using the default value, which was 200ms.

Comment (alvarolaserna) [reply lib/core/device.rb:680]: Hmmmmm, I see, well, tbh, I think thats way too long, probably worth it put it 1 second max

Comment (alvarolaserna) [reply lib/core/device.rb:680]: Okay, so for the sake of keeping the same behaviour as previously, we can keep it like it was. We will have to add an optional value for time in future.

**Review by alvarolaserna — APPROVED** [review]

**Pull request merged**